Sources/MapboxDirections/Intersection.swift

```diff
@@ -323,9 +319,11 @@ extension Intersection: Codable {
             approachLanes = lanes.map { $0.indications }
             usableApproachLanes = lanes.indices { $0.isValid }
             preferredApproachLanes = lanes.indices { ($0.isActive ?? false) }
-            usableLaneIndication = lanes.compactMap { $0.validIndication }[0]
-//            let usableIndications = lanes.compactMap { $0.validIndication }
-//            usableLaneIndication = usableIndications.reduce(ManeuverDirection(rawValue: "none")) { $0.union($1) }
+            if Set(lanes.compactMap { $0.validIndication}).count > 1 {
+                let context = EncodingError.Context(codingPath: decoder.codingPath, debugDescription: "Inconsistent valid indications.")
+                throw EncodingError.invalidValue(Set(lanes.compactMap { $0.validIndication}).count, context)
```

Comment:
Returning the set itself would be more informative than the count of unique indications.